src/foam/u2/view/RichChoiceView.js

```diff
@@ -725,9 +725,12 @@ foam.CLASS({
             'text-overflow': 'ellipsis'
           });
 
-          return this.add(this.fullObject$.map(o => {
-            return ( o && o.toSummary() ) || this.defaultSelectionPrompt;
-          }));
+          if ( this.fullObject ) {
+            var summary = this.fullObject.toSummary();
+            return this.translate(summary, summary);
+          } else {
```

Comment:
No need for the 'else', if you've already returned in the 'if'.

Reply:
@kgrgreer thanks. updated